packages/lpa-questionnaire-web-app/src/controllers/upload-question.js

```diff
@@ -2,22 +2,24 @@ const { VIEW } = require('../lib/views');
 const {
   fileErrorSummary,
   fileUploadNunjucksVariables,
-  deleteFile,
   uploadFiles,
 } = require('../lib/file-upload-helpers');
+const { deleteDocument } = require('../lib/documents-api-wrapper');
 const getAppealSideBarDetails = require('../lib/appeal-sidebar-details');
 const { getTaskStatus } = require('../services/task.service');
 const { createOrUpdateAppealReply } = require('../lib/appeal-reply-api-wrapper');
 
+const docArrayFromInputString = (tempDocsString) => {
+  return tempDocsString
+    ? tempDocsString.match(/{.*?}/g).map((docString) => JSON.parse(docString))
+    : [];
+};
```

Comment:
![Screenshot from 2021-05-28 08-11-39](https://user-images.githubusercontent.com/6329541/119944580-58bc6380-bf8c-11eb-8726-441058fb3399.png)

potentially need a stricter guard before the `match` ? 

https://jsbin.com/hilagugofe/edit?js,console

Reply:
yeah happy to add a type check in there

Reply:
fix is in